.github/workflows/workflow.yaml

```diff
@@ -15,10 +15,8 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Get a specific Node.js version such that hardhat does not hang periodically
```

Comment:
This no longer works, need to get v18.20, and the bug seems to be fixed a long time ago.